lib/view_component/config.rb

```diff
@@ -17,6 +17,7 @@ def defaults
           preview_route: "/rails/view_components",
           show_previews_source: false,
           instrumentation_enabled: false,
+          use_deprecated_instrumentation_name: true,
```

Comment:
I agree this name reads better. I had used the other name so that it would alphabetically show up next to "instrumentation_enabled" in any sort of auto-gened docs.

I just noticed that `instrumentation_enabled` shows in the API.md doc file.  Should I add this key there as well?

Reply:
`api.md` is auto-generated by YARD using the comments from the source code. I think the comments you added are enough.

You can regenerate the docs locally with `bundle exec rake docs:build`.

Reply:
Thanks Spone! I did not know that.

Reply:
> I just noticed that instrumentation_enabled shows in the API.md doc file. Should I add this key there as well?

API.md is auto-generated from YARD comments when we deploy the docs, so that should take care of itself 😄